### ISSUE TITLE
Catch the first touch event to prevent blind clicks on a blanked touchscreen.

### ISSUE THREAD

**Issue by Oquirella** [swaywm/swayidle#142]
I this possible with swayidle or another application on wayland?

**Comment by nicoske** [comment]
I'm wondering if this is possible as well, couldn't figure out how to ignore first touch that should be used to wake up screen. Maybe using swaylock ?

**Comment by nicoske** [comment]
So I had a look with swaylock but there's no way to just ignore the authentication part.
It could be achieved using a very simple wayland application that just catch any input event on the screen then exit..

**Comment by emersion** [comment]
Yeah, this can be achieved with a fullscreen transparent layer-shell program. But it is out-of-scope for swayidle.
